roles/libvirt_manager/tasks/clean_layout.yml

```diff
@@ -22,7 +31,7 @@
       cifmw_libvirt_manager_dependency_packages |
       difference(ansible_facts.packages.keys()) |
       length == 0
-    - _virtqemud.stat.exists
+    - ansible_facts['services']['virtqemud.service']['status'] | default('not-found') != 'not-found'
```

Comment:
Shouldn't it be already passing this conditional, based on [the line modified above](https://github.com/openstack-k8s-operators/ci-framework/pull/2828/files#diff-3193dfb4f3068ac49015aceb156d84037f57116dcd60094d44809a8a9c4eaf68R11)?

Reply:
only if the service exist, same problem if it wasn't installed/deleted for some reason